packages/opal-client/opal_client/config.py

```diff
@@ -123,6 +123,8 @@ def load_policy_store():
 
     OPA_HEALTH_CHECK_POLICY_PATH = 'opa/healthcheck/opal.rego'
 
+    OPAL_CLIENT_SCOPE_ID = confi.str('OPAL_CLIENT_SCOPE_ID', 'default')
```

Comment:
you don't need `OPAL_` prefix here -> not in the var name and not in the string itself -> `Confi` will add these by itself.